__tests__/get.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require("supertest");
```

Comment:
This is absolutely fine BUT maybe overkill to have a separate file for tests for each endpoint (sorry - I should've been more specific). I'd go for something like app.test.js or, if you want to be more specific, maybe do it by table e.g. topics.test.js